README.md

```diff
@@ -312,7 +312,15 @@ config = InitConfig(ef=embedding_functions.OpenAIEmbeddingFunction(
             ))
 naval_chat_bot = App(config)
 
-add_config = AddConfig() # Currently no options
+# Example: define your own chunker config
+add_config = {
```

Comment:
add_config should work at data_type level, so in the example we should show that we are overriding chunking for a particular config and not everything. Otherwise the users will feel consued.

Can you update?

Reply:
So you mean I should only change add_config for one of the data_type (e.g. youtube) instead of all data_types?